crates/bevy_ecs/src/component.rs

```diff
@@ -221,17 +221,18 @@ pub struct ComponentHooks {
 
 impl ComponentHooks {
     /// Register a [`ComponentHook`] that will be run when this component is added to an entity.
-    /// An `on_add` hook will always be followed by `on_insert`.
+    /// An `on_add` hook will always be followed by `on_insert`. Spawning an entity counts as
```

Comment:
What does `"will always be followed by `on_insert`" mean?

Reply:
I think this is just trying to say that these run before on_insert hooks?

Reply:
Yes, it means after `on_add` is ran, `on_insert` is always called also.

Reply:
```suggestion
    /// An `on_add` hook will always run before `on_insert` hooks. Spawning an entity counts as
```